lib/brook.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Brook do
   @type event_type :: String.t()
   @type event :: term()
+  @type author :: String.Chars.t()
```

Comment:
Is this used?

Reply:
It should be a required field of the event message. The `send/3` function that a driver needs to implement to meet its contract takes the author as the second argument.